lib/compress/zstd_compress.c

```diff
@@ -4227,6 +4304,127 @@ static const ZSTD_compressionParameters ZSTD_defaultCParameters[4][ZSTD_MAX_CLEV
 },
 };
 
+static const ZSTD_compressionParameters
```

Comment:
I'm wondering if having dedicated tables is really necessary,
or if it could be derived from the regular table with some local adjustments (such as `+ DD_BLOG`).